.github/workflows/updatecli.yml

```diff
@@ -0,0 +1,36 @@
+name: "Updatecli: Dependency Management"
+
+on:
+  schedule:
+    # Run every hour
+    - cron: '0 * * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  issues: write
+  pull-requests: write
+
+jobs:
+  updatecli:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.19.3'
```

Comment:
Does this affect the go mod output, essentially pinning the go version?

Reply:
Ideally it doesn't affect, but if it makes any unexpected change, we will catch that during PR review.

Reply:
sorry, just basing this on the readme, maybe we should just remove the line about go modules?
"Regular updates in major packages ecosystem, like Go Modules, should be done accordingly to the targeted repo owns definition, e.g., manually or by using Dependabot and/or Renovate."

Reply:
ok, sounds good.